keps/sig-instrumentation/1748-pod-resource-metrics/README.md

```diff
@@ -409,11 +416,198 @@ enhancement:
   CRI or CNI may require updating that component before the kubelet.
 -->
 
+
+## Production Readiness Review Questionnaire
+
+<!--
+
+Production readiness reviews are intended to ensure that features merging into
+Kubernetes are observable, scalable and supportable; can be safely operated in
+production environments, and can be disabled or rolled back in the event they
+cause increased failures in production. See more in the PRR KEP at
+https://git.k8s.io/enhancements/keps/sig-architecture/1194-prod-readiness.
+
+The production readiness review questionnaire must be completed and approved
+for the KEP to move to `implementable` status and be included in the release.
+
+In some cases, the questions below should also have answers in `kep.yaml`. This
+is to enable automation to verify the presence of the review, and to reduce review
+burden and latency.
+
+The KEP must have a approver from the
+[`prod-readiness-approvers`](http://git.k8s.io/enhancements/OWNERS_ALIASES)
+team. Please reach out on the
+[#prod-readiness](https://kubernetes.slack.com/archives/CPNHUMN74) channel if
+you need any help or guidance.
+
+-->
+
+### Feature Enablement and Rollback
+
+* **How can this feature be enabled / disabled in a live cluster?**
+  - [ ] Feature gate (also fill in values in `kep.yaml`)
+    - Feature gate name:
+    - Components depending on the feature gate:
+  - [x] Other
```

Comment:
That's not really disabling it. So, if for example this new endpoint turns out to have a high cost when scraped, and can therefore be used as a DOS attack on the scheduler, there is no way to turn off the endpoint?

Reply:
The cost when scraped is the same as metrics, so the existing behavior for metrics is covered.  We do not allow metrics to be disabled (but if you disable the metrics set pre-GA it's roughly disablement).  You can disable the entire scheduler HTTP server.

I can clarify, but we generally don't make metrics optional and RBAC would be one way of preventing access to metrics, as would be having the scheduler not listen.  Anyone who can DOS the scheduler can do so today using the existing metrics, so we aren't adding a new DOS angle.

Reply:
Ok, can you just say that (disable the scheduler http server) and how to do it?